src/firefly/js/ui/panel/CollapsiblePanel.jsx

```diff
@@ -136,10 +136,13 @@ export function CollapsibleItemView({isOpen, header, title, onToggle, slotProps,
         onToggle?.(v);
     };
 
+    const component= slotProps?.header.component;
+
     return (
         <Accordion {...props} expanded={isOpen} onChange={onChange}>
             <Tooltip {...{title, ...slotProps?.tooltip}}>
                 <AccordionSummary {...slotProps?.header}
+                                    slotProps={ { button: { component } }}
```

Comment:
`component` is a defined prop of AccordionSummary.  By using setting `slotProps.header.component`, you will override the `root's` component and setting the that same value as a prop to the `button` slot.  These are 2 different types.  I don't think that's what you want.